src/batch_processing/metastore_db/log/README_DO_NOT_TOUCH_FILES.txt

```diff
@@ -0,0 +1,8 @@
+
+# *************************************************************************
```

Comment:
You should uncommit these files. It is good that you have already added metastore_db to .gitignore so they are not tracked.

Reply:
@tyaq good call, I got those out